test/message/nexttick_throw.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
`no-undef` is one of the few rules that do something good (as opposed to the most eslint rules that do nothing except for enforcing a ~~religion~~ code style). I don't think it should be disabled for all tests, because it could catch real errors.

What about explicitly declaring globals in every file?

```js
/* globals: undefined_reference_error_maker */
```

I don't remember the exact syntax, but I think you can add globals on per-file basis.